starter-code/index.js

```diff
@@ -1,11 +1,47 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
-};
+  constructor() {
+    // this.items = listItems;
+    // this.length = listLength;
+    this.items = [];
+    // this.length = 0;
+    this.length = this.items.length;
+  }
+  add(x) {
+    this.items.push(x);
+    this.length++;
+  }
+  get(pos) {
+    var position = this.items[pos - 1];
+    return position;
+  }
+  max() {
+    if (this.items.length === 0) {
+      throw new Error("EmptySortedList");
+    }
+    return Math.max(items);
+  }
+  min() {
+    if (this.items.length === 0) {
+      throw new Error("EmptySortedList");
+    }
+    return Math.min(items);
+  }
+  average() {
+    if (this.items.length === 0) {
+      throw new Error("EmptySortedList");
+    }
+  }
+  sum() {
+    var sumsum = 0;
+    if (this.items.length === 0) {
+      throw new Error("EmptySortedList");
```

Comment:
to pass one more test you need to `return 0`  rather than throwing a new Error. The test message is 'should return 0 for an empty sorted list'